docker/Dockerfile.metad

```diff
@@ -7,6 +7,7 @@ RUN cd /home/nebula/BUILD/package \
 
 FROM centos:7
 
+COPY --from=builder /home/nebula/BUILD/build/cpack_output/nebula-*-common.rpm /usr/local/nebula/nebula-common.rpm
```

Comment:
Why need this?

Reply:
Some resource files have been packaged in common rpm.